README.md

```diff
@@ -257,6 +257,9 @@ In the route creation you can override the same settings of the plugin registrat
 - `onExceeding` : callback that will be executed each time a request is made to a route that is rate limited
 - `onExceeded` : callback that will be executed when a user reached the maximum number of tries. Can be useful to blacklist clients
 
+### Examples of Custom Store
+#### [Knex-SQLite](https://github.com/fastify/fastify-rate-limit/blob/master/example/example-knex.js)
+#### [Sequelize-PostgreSQL](https://github.com/fastify/fastify-rate-limit/blob/master/example/example-sequelize.js)
```

Comment:
```suggestion
#### [Knex-SQLite](./example/example-knex.js)
#### [Sequelize-PostgreSQL](./example/example-sequelize.js)
```

Reply:
Made commit as per your suggestion. Thanks @jsumners

Reply:
🤔 GitHub is still showing the `####` instead of `- `.

Reply:
I had only updated the links and missed changing `####` to `- `.
Have fixed this another commit.
Thanks again @jsumners